.github/workflows/ci_cd_pr.yml

```
@@ -80,11 +82,11 @@ jobs:
       pull-requests: write
     steps:
 
-      - uses: ansys/actions/check-pr-title@main
+      - uses: ansys/actions/check-pr-title@feat/check-actions-security-action
```

```suggestion
      - uses: ansys/actions/check-pr-title@main
```
Needs to be reverted before merging